src/c++/perf_analyzer/infer_context.h

```diff
@@ -185,7 +185,10 @@ class InferContext {
   std::function<void(uint32_t)> async_callback_finalize_func_ = nullptr;
 
  private:
-  const RequestRecord::ResponseOutput GetOutput(
+  // Get all the request inputs that are being sent to the server.
```

Comment:
let's comment them all (properly) or omit comments. I prefer to omit. This function does what the name implies, and there are no args.

Reply:
Done